packages/cli/lib/lib/webpack/transform-config.js

```diff
@@ -235,7 +235,9 @@ class WebpackConfigHelpers {
 			)
 			.reduce((arr, loaders) => arr.concat(loaders), [])
 			.filter(
-				({ loader }) => loader === name || (loader && loader.loader === name)
+				({ loader }) =>
+					(typeof loader === 'string' && loader.includes(name)) ||
+					(typeof loader.loader === 'string' && loader.loader.includes(name))
```

Comment:
There very well could be a prettier way to express this but it's Friday and my brain is fried